src/app.jsx

```diff
@@ -91,22 +92,11 @@ class App extends React.Component {
   }
 
   _initTimeline() {
-
     const updateTimelineDates = function(dates) {
-      console.log('timeline dates', moment.utc(dates.to).format());
-      // this.setState({ timelineDates: dates });
-
-      router.update({
-        timelineDate: moment.utc(dates.to).format('YYYY-MM-DD')
-      });
-    };
-
-    const updateMapDates = function (dates) {
-      // this.setState({ mapDates: dates });
-
-      //MAP STATE CHANGE
-      // console.log(dates);
-      // this.mapView.state.set({ timelineDates: dates });
+      console.log('timeline dates recieve', moment.utc(dates.to).format());
+      const date = moment.utc(dates.to).format();
```

Comment:
Be careful with date format, you are showing up a wrong format (it should be .format('YYYY-MM-DD'))